tests/ecall/host/CMakeLists.txt

```diff
@@ -6,6 +6,6 @@ oeedl_file(../ecall.edl host gen)
 add_executable(ecall_host host.cpp ${gen})
 
 target_include_directories(ecall_host PRIVATE ${CMAKE_CURRENT_BINARY_DIR}
-    ${CMAKE_CURENT_SOURCE_DIR})
+    ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Makes me think we don't need this folder at all.

Reply:
So wait do we need this folder or not?

Reply:
I think we do. CMAKE_CURRENT_SOURCE_DIR it is used a lot in openenclave codebase , those 4 were typos missing an R

Reply:
@andschwa I removed the dir variables with typos completely